src/Microsoft.OData.Edm/Vocabularies/CapabilitiesVocabularies.xml

```diff
@@ -744,13 +812,14 @@ supported:
 
       <Term Name="ReadRestrictions" Type="Capabilities.ReadRestrictionsType" AppliesTo="EntitySet Singleton FunctionImport">
         <Annotation Term="Core.Description"
-          String="Restrictions on read operations: retrieve a collection, retrieve a single instance, invoke a function" />
+          String="Restrictions for retrieving a collection of entities, retrieving a singleton instance, invoking a function" />
       </Term>
-      <ComplexType Name="ReadRestrictionsType">

```

Comment:
Did we lose ReadRestrictionsType?   We should have:

ReadRestrictionsBase
ReadRestrictionsType : ReadRestrictionsBase
ReadRestrictionsByKey : ReadRestrictionsBase

Reply:
No. See line 833. We have both.

Reply:
Got it; missed that with all of the highlighting. Thanks.

---
In reply to: [268340091](https://github.com/OData/odata.net/pull/1428#discussion_r268340091) [](ancestors = 268340091)